webapp/model/products.json

```
@@ -0,0 +1,106 @@
+{  
+    "Products": [
+        {
+            "Id": "1",
+            "Name": "tea",
+            "Description": "very useful thing",
+            "Price": 2.65,
+            "Quantity": "134",
+            "Supplier": ["1", "2"],
+            "Category": "relax",
+            "Availability": true
+        },
+        {
+            "Id": "2",
+            "Name": "coffee",
+            "Description": "very cheerful thing",
+            "Price": 21.65,
+            "Quantity": "22",
+            "Supplier": ["1", "3"],
+            "Category": "enjoy",
```

Product's category should not be name but id, so in this line you should change it to:
```suggestion
            "Category": "2",
```